tests/callbacks/test_device_stats_monitor.py

```diff
@@ -85,6 +102,128 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
     trainer.fit(model)
 
 
+@RunIf(min_gpus=1)
```

Comment:
```suggestion
@RunIf(min_gpus=1, min_torch="1.8")
```
although, we run with 1.8 on ci, but this should not fail if someone's running it on their personal machine.